packages/sit-onyx/src/components/OnyxNavItem/OnyxNavItem.vue

```diff
@@ -41,6 +48,7 @@ const { t } = injectI18n();
     @keydown.enter="props.href && emit('navigate', props.href)"
   >
     <span>{{ props.label }}</span>
+    <OnyxIcon v-if="shouldShowExternalIcon" :icon="arrowSmallUpRight" size="16px" />
```

Comment:
Imho the icon should be top aligned like in the link component

I left a Figma comment for Jannick